PerfettoCds/Pipeline/DataOutput/PerfettoGenericEvent.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Microsoft.Performance.SDK;
+using System.Collections.Generic;
+
+namespace PerfettoCds.Pipeline.DataOutput
+{
+    /// <summary>
+    /// A generic app/component event that contains event name, event metadata, and thread+process
+    /// info.
+    /// </summary>
+    public class PerfettoGenericEvent
```

Comment:
might be good to change this to a readonly struct to help memory

Reply:
Sure thing